src/codec.rs

```diff
@@ -552,6 +552,55 @@ impl<T: Decode> Decode for Option<T> {
 	}
 }
 
+const _: () = {
```

Comment:
Please move the import to the top and I don't understand why you want to wrap it into an unnamed const.

Reply:
I did this to have the import statement for the `NonZero*` types close to their use site. I generally prefer this in code bases but will change it for this PR.

Reply:
done